src/sankey.js

```diff
@@ -209,30 +250,32 @@ export default function Sankey() {
   }
 
   function initializeNodeBreadths(columns) {
-    const ky = min(columns, c => (y1 - y0 - (c.length - 1) * py) / sum(c, value));
+    const ky = min(columns, c => (Math.abs(y1 - y0) - (c.length - 1) * py) / sum(c, value));
     for (const nodes of columns) {
-      let y = y0;
+      let ystart = orient === bottom ? y1 : y0;
```

Comment:
Suggest camelCase for `yStart` and `yEnd`.
```suggestion
      let yStart = orientation === bottom ? y1 : y0;
```

Reply:
Sounds good!

NB: I divided this into two separate commits to keep the variable renaming in separate commits (one for `orient` -> `orientation`, and one for the `ystart`, `yend` variables).